aiida/cmdline/commands/data/bands.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida_core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+import sys
+import click
+from aiida.backends.utils import load_dbenv, is_dbenv_loaded
+from aiida.cmdline.commands.data.list import _list, list_options
+from aiida.cmdline.commands.data.export import _export
+from aiida.cmdline.params.options.multivalue import MultipleValueOption
+from aiida.cmdline.commands import verdi_data
+from aiida.cmdline.params import arguments
+from aiida.cmdline.params import options
+from aiida.cmdline.utils import echo
+from aiida.orm.data.array.bands import BandsData
+from aiida.common.utils import Prettifier
+
+# if not is_dbenv_loaded():
```

Comment:
remove unused comments, especially code like this, which would potentially slow down all of verdi.

Reply:
done